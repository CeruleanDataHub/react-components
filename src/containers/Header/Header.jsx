import React from "react";
import styled from "styled-components";
import PropTypes from "prop-types";

const TopHeader = styled.header`
  position: fixed;
  height: 0;
  color: #ffffff;
  z-index: 1;
`;

const Logo = styled.img`
  width: 10em;
  margin-left: 4em;
`;

export const Header = ({ logo }) => (
  <TopHeader id="header" image={logo}>
    <Logo src={logo} />
  </TopHeader>
);

Header.propTypes = {
  logo: PropTypes.node
};

Header.defaultProps = {
  logo: null
};
