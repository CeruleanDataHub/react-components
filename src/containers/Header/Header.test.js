import React from "react";
import { mount } from "enzyme";

import { Header } from "./Header";

describe("Header", () => {
  let component;

  beforeEach(() => {
    component = mount(<Header />);
  });

  it("renders", () => {
    expect(component).toMatchHtmlSnapshot();
  });
});
