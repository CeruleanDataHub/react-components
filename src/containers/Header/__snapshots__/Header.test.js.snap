// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<header
  class="sc-AxjAm iYZhvh"
  id="header"
>
  <img
    class="sc-AxirZ bmeYGH"
  />
</header>
`;
